src/plugins/intel_cpu/src/utils/fusing_filter.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
should be 2018-2022 I guess

Reply:
In general the rule is to have first year as file creation date and the second year as last modified / released date.
But the initial / creation date should not be changed for sure, this is the date when file copyright is claimed to start from, not a copyright of the whole project.
Will update to 2021-2022
Let me know if we have some other rules

Reply:
Updated to 2021-2022